### ISSUE TITLE
STDPMechanism, not working with nest

### ISSUE THREAD

**Issue by JoelChavas** [NeuralEnsemble/PyNN#308]
in examples, by running:
```
python simple_STDP2.py nest
```
At the call to Projection, we have got the error:
```
 File "/root/PyNN-0.8/pyNN/nest/projections.py", line 170, in _identify_common_synapse_properties
    local_parameters = nest.GetStatus(connections)[0].keys()
```
It corresponds to the fact that the call to `nest.GetConnections(source=[3],target=[6],synapse_model=self.nest_synapse_model)` returns `[]`

Changing the stdp_model STDP_Mechanism into:
```stdp_model=StaticSynapse(weight=0.1)```
removes the error

**Comment by apdavison** [comment]
This problem arises when using `iaf_psc_exp_ps` rather than `iaf_psc_exp` (see https://github.com/nest/nest-simulator/issues/85)

As a workaround, use
```
setup(spike_precision="on_grid")
```
which causes PyNN to choose the non-precise model.

**Comment by apdavison** [comment]
https://github.com/nest/nest-simulator/issues/85 is now closed; the fix is in NEST 2.10.
